bedrock/mozorg/templates/mozorg/diversity/2021/what-we-build.html

```diff
@@ -0,0 +1,350 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "mozorg/diversity/2021/index.html" %}
+
+{% block content %}
+
+
+
```

Comment:
**nit (blocking)**: looks like the images for the cards on this page aren't the correct aspect ratio - i think you mentioned something about this earlier this week - but I don't remember exactly what was the plan moving forward